lib/timestamp.rb

```diff
@@ -0,0 +1,17 @@
+require 'date'
+
+module Timestamp
```

Comment:
This is interesting to me that you've used a module for the timestamp as this is something that I'm not familiar wit yet.